Dockerfile

```diff
@@ -10,7 +10,7 @@ ADD . /opt/crystal-head
 
 WORKDIR /opt/crystal-head
 ENV CRYSTAL_CONFIG_VERSION HEAD
-ENV CRYSTAL_CONFIG_PATH lib:libs:/opt/crystal-head/src:/opt/crystal-head/libs
+ENV CRYSTAL_CONFIG_PATH lib:/opt/crystal-head/src:/opt/crystal-head/libs
```

Comment:
@asterite question, what is inside `/opt/crystal-head/libs`? Perhaps that can be removed too?